dependencies/common/install-boost

```diff
@@ -65,19 +88,24 @@ then
      SYS_MAJOR_VER=`uname -r | cut -d "." -f -1`
      if [ "$SYS_MAJOR_VER" -gt "12" ]
      then
-        CXX_STDLIB_FLAGS=cxxflags="-stdlib=libstdc++"
+        CXX_STDLIB_FLAGS="cxxflags=\"-Wno-unused-variable -stdlib=libstdc++\" linkflags=\"-stdlib=libstdc++\""
      else
-        CXX_STDLIB_FLAGS=""
+        CXX_STDLIB_FLAGS="cxxflags=\"-Wno-unused-variable\""
      fi
 
      sudo ./bjam ${BOOST_BJAM_FLAGS} --prefix=$BOOST_DIR toolset=clang $CXX_STDLIB_FLAGS variant=release threading=multi link=static install
    else
      sudo ./bjam ${BOOST_BJAM_FLAGS} --prefix=$BOOST_DIR variant=release install
    fi
 
+   # rename libraries in the boost install dir
+   cd $BOOST_DIR/lib
+   for file in librstudio*; do
+      sudo mv $file ${file/rstudio_/}
```

Comment:
The boost libraries are generated with names e.g. `librstudio_boost*`, so this code just removed the `rstudio_` bit.